services/newsfeed/package.json

```diff
@@ -33,7 +33,7 @@
     "bluebird": "^3.7.2",
     "html-entities": "^2.3.2",
     "html-to-text": "^8.0.0",
-    "knex": "^2.0.0",
+    "knex": "^2.3.0",
```

Comment:
I think we can remove `knex` as a dependency as we are not using it directly. The current newsfeed implementation is directly using MySQL from framework.

Reply:
Addressed!